templates/_includes/footer.html

```diff
@@ -28,16 +23,14 @@
 {%- endif -%}
 <script src="{{site.url}}/assets/js/nav-accordion.js"></script>
 <script src="https://code.jquery.com/ui/1.12.0/jquery-ui.js"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 <script src="{{site.url}}/assets/js/lightbox.js"></script>
 {%- if page.url contains '/components/'-%}
   {# {%- assign urlStub == {{ site.data.families | jsonify }} -%}
   {%- unless page.url == '{{site.url}}/components/{{urlStub.[0]}}.html' -%} #}
     <script src="{{site.url}}/assets/js/wayfinding.js"></script>
-    <script src="{{site.url}}/assets/js/tooltip.js"></script>
   {# {%- endunless -%} #}
 {%- endif -%}
-<script src="https://2d2bf231b82dfe76fe36-fa12562cfe810d69bedcc36a0ac289ef.ssl.cf1.rackcdn.com/javascript/helix-ui.js"></script>
+<script src="{{site.url}}/node_modules/helix-ui/dist/scripts/helix-ui.browser.min.js"></script>
```

Comment:
Missing additional javascript for HelixUI javascript to work in legacy browsers. 

https://github.com/rackerlabs/helix-ui/blob/master/docs/components/layouts/vertical-layout-template.html#L100-L116